build-and-push.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+eval $(crc console --credentials | grep "admin, run" | cut -f2 -d"'") --insecure-skip-tls-verify=true
+
+
+REGISTRY=$(oc registry info)
+TOKEN=$(oc whoami -t)
+INTERNAL_REGISTRY=$(oc registry info --internal=true)
+buildah login --tls-verify=false -u openshift -p "${TOKEN}" "${REGISTRY}"
+buildah bud -f build/Dockerfile -t "${REGISTRY}/service-telemetry/service-telemetry-operator:latest" .
+buildah push --tls-verify=false "${REGISTRY}/service-telemetry/service-telemetry-operator:latest"
+
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: smartgateway-operator
+  namespace: service-telemetry
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: smartgateway-operator
+  source: infrawatch-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: smart-gateway-operator.v2.0.0
+EOF
+
+oc delete servicetelemetry stf-default
+
+CSV_VERSION=1.1.0
```

Comment:
Can we make this as a parameter?

Reply:
yes, sure.

Reply:
If you just source the `metadata.sh` file then you'll get a bunch of parameters. You'll need to adjust this line depending on your relative position, but it already has the `CSV_VERSION` defined and parameterized.

```
REL=$(dirname "$0"); source "${REL}/../build/metadata.sh"
```